mappings/net/minecraft/inventory/ContainerLock.mapping

```diff
@@ -11,11 +11,7 @@ CLASS net/minecraft/class_1273 net/minecraft/inventory/ContainerLock
 		COMMENT or if the key string is empty.
 		ARG 1 stack
 			COMMENT the key item stack
-	METHOD method_5473 fromTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_1273;
-		COMMENT Creates a new {@code ContainerLock} from the {@code Lock} key of the compound tag.
-		COMMENT <p>
-		COMMENT If the {@code Lock} key is not present, returns an empty lock.
-		ARG 0 tag
-	METHOD method_5474 toTag (Lnet/minecraft/class_2487;)V
-		COMMENT Inserts the key string of this lock into the {@code Lock} key of the compound tag.
-		ARG 1 tag
```

Comment:
hmm enigma must be tripping balls right now

Reply:
uhhhhh does enigma just nuke comments when you rename a method then?

Reply:
that is super annoying

Reply:
I guess make note of all the times that happens and I will fix them all manually like tomorrow

Reply:
See FabricMC/Enigma#190.